betty/jinja2/__init__.py

```diff
@@ -214,6 +192,40 @@ def _init_i18n(self) -> None:
         )
         self.policies['ext.i18n.trimmed'] = True
 
+    @property
+    def context_class(self) -> type[Context]:  # type: ignore[override]
+        # Alias the environment, because `self` is bound to a different instance
+        # inside the class.
+        _environment = self
```

Comment:
Remove this and inject the environment instead. Then the context class may be moved outside it's containing class

Reply:
Or maybe not